### PR TITLE
chore: updates to the kubernetes part documentation of private locations

### PR DESCRIPTION
**Pull request by notdryft**
Motivation:

- Some defaults are "wrong" and/or confusing
- Defaults should be commented

Modification:

- Add missing namespace in Deployment example
- Comment optional configuration
- Change defaults of optional configuration (Xmx8G -> Xmx4G, etc.)

### CODE REVIEW

**Review by Isammoc — APPROVED** [review]

File: content/reference/admin/private_locations/configuration/kubernetes/index.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ control-plane {
       # They are hosted on Docker Hub, and available for the linux/amd64 and linux/arm64 platforms
       image {
         type = certified
-        java = latest # Possible values : 8, 11, 17 or latest
+        java = latest # Possible values : 8, 11, 17, 21 or latest
```
Comment (tpetillot) [inline content/reference/admin/private_locations/configuration/kubernetes/index.md:51]: Need to update and release https://github.com/gatling/frontline-cloud/blob/dev/back/commons/primitive/src/main/scala/io/gatling/enterprise/commons/primitive/CertifiedDockerImage.scala#L12 first

**Review by tpetillot — CHANGES_REQUESTED** [review]

File: content/reference/admin/private_locations/configuration/kubernetes/index.md (hunk 1 above)
Comment (notdryft) [reply content/reference/admin/private_locations/configuration/kubernetes/index.md:51]: oh right

File: content/reference/admin/private_locations/configuration/kubernetes/index.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ control-plane {
       #   "--add-opens=java.base/jdk.internal.misc=ALL-UNNAMED"
       # ]
       # Based on your instance configuration, you may want to update Xmx and Xms values.
-      jvm-options = ["-Xmx8G", "-Xms512M"]
+      # jvm-options = ["-Xmx4G", "-Xms512M"]
```
Comment (slandelle) [inline content/reference/admin/private_locations/configuration/kubernetes/index.md:118]: Do we provide default values? If not, IIRC, Xmx is the smallest of 1/4th of the physical memory or 1GB. Not sure what `physical memory` is on k8s: mem request or mem limit? I'm concern we'd end up with 256MB or less. Shouldn't we set 1GB like on Gatling OSS?

**Review by slandelle — CHANGES_REQUESTED** [review]

File: content/reference/admin/private_locations/configuration/kubernetes/index.md (hunk 2 above)
Comment (tpetillot) [reply content/reference/admin/private_locations/configuration/kubernetes/index.md:118]: Default configuration available at https://github.com/gatling/frontline-cloud/blob/dev/back/hexagon/run/port/model/src/main/scala/io/gatling/enterprise/hexagon/run/model/HardcodedConfiguration.scala#L14

**Review by tpetillot — APPROVED** [review]

**Review by tpetillot — APPROVED** [review]

**Pull request merged**